config/initializers/geocoder.rb

```diff
@@ -0,0 +1,23 @@
+Geocoder.configure(
+  # geocoding options
+  # :timeout      => 3,           # geocoding service timeout (secs)
+   #:lookup       => :bing,     # name of geocoding service (symbol)
+  :lookup       => :google,  
```

Comment:
Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.<br>Trailing whitespace detected.